.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Build ICU
+on:
+  release:
+    types: [published]
```

Comment:
May be omitted in case you want to rely purely on tags. It will create the Github Release for you.